src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -102,18 +100,21 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
 
         if (triggerAction == TDTM_Runnable.Action.AfterInsert &&
             !recursionAfterInsertCheck) {
-                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, accountId);
+                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, NULL);
             }
 
         if (triggerAction == TDTM_Runnable.Action.AfterUpdate &&
             !recursionAfterUpdateCheck) {
-                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, accountId);
+                //Retrieves a map of AccountIds to the number of Contacts associated to the
+                //Account that is not marked deceased or Excluded_from_Household_Naming__c
+                accountIdNumOfCons = retrieveAccountIds(newContacts);
+                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, accountIdNumOfCons);
             }
 
         if (triggerAction == TDTM_Runnable.Action.AfterDelete &&
             !recursionAfterDeleteCheck) {
-                accountId = retrieveAccountId(oldContacts);
-                handlesAfterDelete(oldContacts, accountId);
+                accountIdNumOfCons = retrieveAccountIds(oldContacts);
```

Comment:
NIT: the curly braces/indent here look like maybe they're indented too far? Or was the intention to line the curly brace up with the parameters list? I think in most cases we prefer to line the closing brace up with the "if" (maybe it's just github doing something odd?)

Reply:
@lifewithryan , yep you are correct. Since I've been trying to get into the habit of adding space after if, I've also accidentally been doing it after curly braces.